operations/docker/php72-integrations/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Dockerfile
+FROM circleci/php:7.2-apache-node-browsers-legacy
+
+USER root
+ENV LANG=C.UTF-8
+
+RUN MAGENTO_VERSION=2.3.0
+
+RUN apt-get update && apt-get -y install curl mysql-client libmcrypt-dev mcrypt libpng-dev libjpeg-dev libxml2-dev libxslt-dev
+RUN pecl channel-update pecl.php.net
+RUN pecl install zip &&  docker-php-ext-enable zip
+RUN docker-php-ext-enable xdebug
+RUN docker-php-ext-configure gd --with-jpeg-dir=/usr/include/
+RUN docker-php-ext-install gd
+RUN docker-php-ext-install soap
+RUN docker-php-ext-install xsl
+RUN apt-get install -y libmcrypt-dev
+RUN pecl install mcrypt-1.0.2
+RUN docker-php-ext-enable mcrypt
+RUN docker-php-ext-install bcmath && docker-php-ext-enable bcmath
+RUN docker-php-ext-install pdo_mysql && docker-php-ext-enable pdo_mysql
+RUN composer self-update -q
+
+COPY auth.json /home/circleci/.composer/auth.json
+USER circleci
+WORKDIR /home/circleci
+RUN composer create-project --repository-url=https://repo.magento.com/ magento/project-community-edition=2.3.0 magento/
+WORKDIR /home/circleci/magento
+RUN composer install
+
+# RUN php bin/magento setup:install -q \
+#     --language="en_US" \
+#     --timezone="UTC" \
+#     --currency="USD" \
+#     --db-host=127.0.0.1 \
+#     --db-user=root \
+#     --base-url="http://magento2.test/" \
+#     --admin-firstname="Dev" \
+#     --admin-lastname="Bolt" \
+#     --backend-frontname="backend" \
+#     --admin-email="admin@example.com" \
+#     --admin-user="admin" \
+#     --use-rewrites=1 \
+#     --admin-use-security-key=0 \
+#     --admin-password="123123q"
+
+COPY auth.json /home/circleci/magento/auth.json
```

Comment:
add a comment here on how one would get this auth.json if we needed to rebuild this image

Reply:
Again addressed in this PR https://github.com/BoltApp/bolt-magento2/pull/343